android/app/src/main/java/org/uwaterloo/subletr/pages/home/map/HomeMapChildView.kt

```diff
@@ -42,33 +92,155 @@ fun HomeMapChildView(
 		}
 	}
 
-	LazyColumn(
-		modifier = modifier,
-		content = {
-			// TODO: Remove in future, for testing
```

Comment:
Where did the todo go ... did we remove this?

Reply:
Yup, we're actually use the page now WOWOOWOWOW